packages/integrations/node/test/api-route.test.js

```diff
@@ -106,6 +107,6 @@ describe('API routes', () => {
 
 		await done;
 
-		expect(locals).to.deep.include({ cancelledByTheServer: true });
+		assert.notStrictEqual(locals, { cancelledByTheServer: true });
```

Comment:
And this. Should it be `assert.equal(locals.cancelledByTheServer, true)`?

Reply:
Unfortunately, there's currently no way of asserting that _actual_ contains _expected_ using `node:assert`.
This one can be changed to something like `assert.strictEqual(locals.cancelledByTheServer, true)`. For more complex examples, a helper function that checks entries is probably the best we can do.